src/FileSavers/ProjectDescriptionSaver.cs

```diff
@@ -353,6 +356,16 @@ private static void SaveData(string pathToFile, StringBuilder fileData)
             currentVersion[eplannerVersionId] = string.Empty;
 
             bool save = !currentVersion.SequenceEqual(previousVersion);
+
+            long fileSpace = currentVersion.Length - previousVersion.Length;
+            if (save && fileSpace > freeSpace * 3)
```

Comment:
Тут "3" лучше выделить в константу:
```suggestion
            if (save && fileSpace > freeSpace * 3)
```